src/api/controllers/articleTagController.js

```diff
@@ -0,0 +1,51 @@
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+import errorMessages from '../../helpers/constants/error.messages';
+// import errorMessages from '../../helpers/constants/error.messages';
```

Comment:
Ehhh @nkpremices remove this comment codes please

Reply:
Cool bro